packages/clay-card/src/CardWithFile.tsx

```diff
@@ -0,0 +1,152 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayCard from './Card';
+import ClayForm from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import ClaySticker from '@clayui/sticker';
+import React from 'react';
+import {ClayDropDownWithBasicItems} from '@clayui/drop-down';
+
+interface IProps {
+	/**
+	 * List of actions in the dropdown menu
+	 */
+	actions?: React.ComponentProps<typeof ClayDropDownWithBasicItems>['items'];
+
+	/**
+	 * Description of the file
+	 */
+	description?: React.ReactText;
+
+	/**
+	 * Path or URL to file
+	 */
+	href?: string;
+
+	/**
+	 * List of labels that are applied to the file
+	 */
+	labels?: Array<
+		React.ComponentProps<typeof ClayLabel> & {value: React.ReactText}
+	>;
+
+	/**
+	 * Name of the file
+	 */
+	name: string;
+
+	/**
+	 * Callback for when item is selected
+	 */
+	onSelectChange?: (val: boolean) => void;
+
+	/**
+	 * Flag to indicate if card is selected
+	 */
+	selected?: boolean;
+
+	/**
+	 * Path to clay icon spritemap
+	 */
+	spritemap: string;
+
+	/**
+	 * Values used in displaying bottom-left icon
+	 */
+	iconProps?: {
+		content: React.ReactNode;
+		displayType: React.ComponentProps<typeof ClaySticker>['displayType'];
+	};
+}
+
+export const ClayCardWithFile: React.FunctionComponent<IProps> = ({
+	actions,
+	description,
+	href,
+	iconProps,
+	labels,
+	name,
+	onSelectChange,
+	selected = false,
+	spritemap,
+}) => {
+	const headerContent = (
+		<ClayCard.AspectRatio className="card-item-first">
+			<div className="aspect-ratio-item aspect-ratio-item-center-middle aspect-ratio-item-fluid card-type-asset-icon">
+				<ClayIcon spritemap={spritemap} symbol="documents-and-media" />
```

Comment:
We will have to make `symbol` configurable here, the user or the product that using can modify it.

Reply:
Hmm, what would you imagine users changing it to? I was looking in the docs and it seemed explicit that this was the symbol someone would see.

Reply:
The icons may be different, so we as a library should let the product team set this up easily without holding them back so much that they have to issue an issue here just to change the icon I want to leave this part of the high-level not too rigid in order not to reach that level, to certain things that we can make configurable.

But in the documentation has the description of making this configurable, are you looking at this document https://docs.google.com/document/d/1st2moaRVphrsFrDJUz-ic26aH-V46YkQam75X6BHPY0/edit#heading=h.eevh9ngq3hi8?

Reply:
ah, I wasn't able to find that doc. that should help. thanks